index.js

```diff
@@ -479,10 +479,10 @@ module.exports = {
 				array: 'Array',
 				Boolean: 'boolean',
 				Integer: 'integer',
-				Number: 'number',
+				number: 'Number',
 				'function': 'Function',
 				object: 'Object',
-				String: 'string'
+				string: 'String'
```

Comment:
Doesn't seem to match the intention of lower case for primitives...

[MDN primitives](https://developer.mozilla.org/en-US/docs/Glossary/Primitive):
> A primitive (primitive value, primitive data type) is data that is not an object and has no methods. In JavaScript, there are 6 primitive data types: string, number, boolean, null, undefined, symbol (new in ECMAScript 2015).

Potentially confusing because there are Object wrappers for the primitives: Number, String and Symbol.